Simplenote/TagListViewController+Swift.swift

```diff
@@ -87,6 +112,12 @@ extension TagListViewController {
         let absoluteOffSetY = scrollView.documentVisibleRect.origin.y + clipView.contentInsets.top
         return min(max(absoluteOffSetY / SplitItemMetrics.headerMaximumAlphaGradientOffset, 0), 1)
     }
+
+    private func notifyTagsListFilterDidChange() {
+        let isViewingTrash = selectedNotesFilter == .deleted
+        let name: NSNotification.Name = isViewingTrash ? .TagListDidBeginViewingTrash : .TagListDidBeginViewingTag
```

Comment:
Wondering if we need separate notifications here? 🤔

Reply:
Can be definitely simplified, but since we had to patch the Editor as well, I kept the change as small as possible.

I'll be revisiting this, if FRC goes as expected, the TagList should get the same refactor! 🔥